app/models/foreman_kubevirt/kubevirt.rb

```diff
@@ -284,6 +284,25 @@ def host_interfaces_attrs(host)
       end
     end
 
+    #
+    # Overrding base class implementation since 'pvc' is required
+    #
+    def set_vm_volumes_attributes(vm, vm_attrs)
+      volumes = vm.volumes.select { |v| v.type == 'persistentVolumeClaim' } || []
+      vm_volumes = []
+      volumes.each do |vol|
+        begin
+          vol.pvc = client.pvcs.get(vol.info)
+          vm_volumes << vol
+        rescue Exception => e
+          # There is an option we attempt to import a VM that its PVC doesn't exist
```

Comment:
In that case, I think the user should have an option to add PVC.

Reply:
* We don't support update of an existing VM from foreman.
* There is a possibility that the vm is based on an image (containerDisk) which isn't shown on the UI under the Host --> VM --> Storage, in this case no storage is needed.